odl/solvers/smooth/newton.py

```diff
@@ -34,6 +34,54 @@
 # TODO: update all docs
 
 
+def _bfgs_multiply(s, y, x, hessinv_estimate=None):
+    """Computes ``Hn^-1(x)`` for the L-BFGS method.
+
+    Parameters
+    ----------
+    s : `sequence` of `LinearSpaceElement`
+        The ``s`` coefficients in the BFGS update, see Notes.
+    y : `sequence` of `LinearSpaceElement`
+        The ``y`` coefficients in the BFGS update, see Notes.
+    x : `LinearSpaceElement`
+        Point in which to evaluate the product.
+    hessinv_estimate : `Operator`, optional
+        Initial estimate of the hessian ``H0^-1``.
```

Comment:
What happens if `hessinv_estimate` is not give? Taken as identity?

Edit: I guess this is only called from the bfgs method, and there it says that identity operator is default. Maybe that is enough...